### ISSUE TITLE
regrtest: re-run failed tests in subprocesses

### ISSUE THREAD

**Issue by vstinner** [python/cpython#81892]
BPO | [37711](https://bugs.python.org/issue37711)
--- | :---
Nosy | @vstinner

<sup>*Note: these values reflect the state of the issue at the time it was migrated and might not reflect the current state.*</sup>

<details><summary>Show more details</summary><p>

GitHub fields:
```python
assignee = None
closed_at = None
created_at = <Date 2019-07-29.23:44:36.211>
labels = ['tests', '3.9']
title = 'regrtest: re-run failed tests in subprocesses'
updated_at = <Date 2019-07-29.23:45:04.178>
user = 'https://github.com/vstinner'
```

bugs.python.org fields:
```python
activity = <Date 2019-07-29.23:45:04.178>
actor = 'vstinner'
assignee = 'none'
closed = False
closed_date = None
closer = None
components = ['Tests']
creation = <Date 2019-07-29.23:44:36.211>
creator = 'vstinner'
dependencies = []
files = []
hgrepos = []
issue_num = 37711
keywords = []
message_count = 1.0
messages = ['348700']
nosy_count = 1.0
nosy_names = ['vstinner']
pr_nums = []
priority = 'normal'
resolution = None
stage = None
status = 'open'
superseder = None
type = None
url = 'https://bugs.python.org/issue37711'
versions = ['Python 3.9']
```

</p></details>

**Comment by vstinner** [comment]
While it would be nice, I don't have the bandwidth to work on this issue, so I just close it.

**Comment by vstinner** [comment]
The feature was implemented in issue #108834.
